pomm-project/pomm-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "alias": "pomm",
```

Comment:
I think aliases are not allowed in the contrib repository.

@javiereguiluz / @fabpot ?